run_dir/static/css/status_b5.css

```diff
@@ -83,6 +83,15 @@ tfoot {
 .tt-suggestion p { margin: 0; }
 .tt-hint { visibility:hidden; }
 
+/* Project Meta Comparison search badges */
+
+#badges .badge {
+  font-size: 1em;
+}
+
+#badges .badge:hover {
```

Comment:
Maybe a more specific name than just `badges`?

Reply:
Sure, I was thinking of it, but it shouldn't be too long 🤔 

`#pid_badges`? `#pid_del_badges`?

Reply:
More descriptive the better. It doesn't matter if its a bit longer, as long as its easily understandable.

Reply:
Then I'll go with `#del_pid_badges` 👍🏻